Telegram bot

```diff
@@ -0,0 +1,29 @@
+import logging
+from telegram.ext import Updater, CommandHandler, MessageHandler
+
+TOKEN = 7901866660:AAFJ2Te56IAreu3nxnXKgRrQXNEKMF3CoSg  # 
```

Comment:
you should NEVER give publicly any credentials, this violates point 4.5 of the Telegram Bot Platform Developer Terms of Service (https://telegram.org/tos/bot-developers#4-5-credentials)